pylops/waveeqprocessing/kirchhoff.py

```diff
@@ -242,28 +245,28 @@ class Kirchhoff(LinearOperator):
 
     def __init__(
         self,
-        z,
-        x,
-        t,
-        srcs,
-        recs,
-        vel,
-        wav,
-        wavcenter,
-        y=None,
-        mode="eikonal",
-        wavfilter=False,
-        dynamic=False,
-        trav=None,
-        amp=None,
-        aperture=None,
-        angleaperture=90,
-        anglerefl=None,
-        snell=None,
-        engine="numpy",
-        dtype="float64",
-        name="D",
-    ):
+        z: NDArray,
+        x: NDArray,
+        t: NDArray,
+        srcs: NDArray,
+        recs: NDArray,
+        vel: Union[float, NDArray],
+        wav: NDArray,
+        wavcenter: int,
+        y: Optional[NDArray] = None,
+        mode: str = "eikonal",
+        wavfilter: bool = False,
+        dynamic: bool = False,
+        trav: Optional[NDArray] = None,
+        amp: Optional[NDArray] = None,
+        aperture: Optional[Union[float, Tuple]] = None,
+        angleaperture: Union[float, Tuple] = 90,
+        anglerefl: Optional[NDArray] = None,
+        snell: Optional[Union[float, Tuple]] = None,
```

Comment:
If possible, identify what the `Tuple` is of. For example, a `Tuple` of two ints: `Tuple[int, int]`. Or a `Tuple` of an unknown number of floats: `Tuple[float, ...]`.